base/broadcast.jl

```diff
@@ -267,9 +267,9 @@ instantiate(x) = x
 
 Construct and check the axes for the lazy Broadcasted object `bc`.
 
-Custom `BroadcastStyle`s may override this default in cases where it is fast and easy
+Custom [`BroadcastStyle`](@ref)s may override this default in cases where it is fast and easy
```

Comment:
I don't think there are docs for `BroadcastStyle`.

Reply:
https://github.com/JuliaLang/julia/blob/master/base/broadcast.jl#L112 ?
Appears in the docs: https://docs.julialang.org/en/latest/base/arrays/#Base.Broadcast.BroadcastStyle

Reply:
Oh, okay. Never mind then.